doc/man/8/ceph.rst

```diff
@@ -375,13 +375,13 @@ Subcommand ``deactivate`` stops mds.
 
 Usage::
 
-	ceph mds deactivate <who>
+	ceph mds deactivate <role>
 
 Subcommand ``fail`` forces mds to status fail.
 
 Usage::
 
-	ceph mds fail <who>
+	ceph mds fail <role>
```

Comment:
`mds fail` is unfortunately special because it can also take a `gid` argument in addition to a role. Look at this:

https://github.com/ceph/ceph/blob/master/src/mon/MDSMonitor.cc#L1140

We should use `mds fail <role|gid>` in the documentation.